src/widget/sidebar/extra.c

```diff
@@ -551,8 +616,11 @@ static void draw_extra_info_buttons(void)
     }
     if (data.info_to_display & SIDEBAR_EXTRA_DISPLAY_REQUESTS && data.active_requests) {
         for (int i = 0; i < data.visible_requests; i++) {
-            button_border_draw(data.x_offset + 2, data.request_buttons_y_offset + i * EXTRA_INFO_HEIGHT_REQUESTS_PANEL,
-                data.width - 4, EXTRA_INFO_HEIGHT_REQUESTS_PANEL, i == data.focused_request_button_id - 1);
+
+            int y_offset = data.request_buttons_y_offset + i * EXTRA_INFO_HEIGHT_REQUESTS_PANEL + 28;
```

Comment:
You'd need to change a bit of code here as well to make the "View all" larger and vertically centered.

Reply:
Yes you can modify